resources/public/api/conf/2.0/common/errors.yaml

```diff
@@ -354,6 +354,12 @@ components:
         code: RULE_DUPLICATE_ACCOUNT_NAME
         message: Duplicate account name given for supplied NINO
 
+    ruleOutsideAmendmentWindow:
+      description: The request cannot be completed as you are outside the amendment window.
+      value:
+        code: RULE_OUTSIDE_AMENDMENT_WINDOW
+        message: You are outside the amendment window
+
```

Comment:
This has been added in a second time, when you preview the OAS this sets all error scenarios to null